src/components/import/ImportTemplateModal.tsx

```diff
@@ -225,7 +226,12 @@ export function ImportTemplateModal(props: Props) {
 
           <Button
             behavior="button"
-            onClick={() => setIsSaveTemplateModalOpen(true)}
+            onClick={() => {
+              setIsTemplateModalOpen(false);
+              setTimeout(() => {
+                setIsSaveTemplateModalOpen(true);
+              }, 310);
```

Comment:
Is there reason why 310?

Reply:
@beganovich The only reason for it is actually that I want to make sure everything is unmounted from the first modal. We have some transitions of 200 and 300 milliseconds in the Modal base component, so I want to make sure that everything is correctly unmounted before opening a second modal. That is the only reason.